kv/tikv/storage/kvstore/transaction.go

```diff
@@ -1,20 +1,126 @@
 package kvstore
 
 import (
+	"bytes"
+	"encoding/binary"
 	"github.com/pingcap-incubator/tinykv/kv/tikv/dbreader"
 	"github.com/pingcap-incubator/tinykv/kv/tikv/inner_server"
 )
 
-// Txn represents an internal transaction (see tikv/storage/doc.go for a definition). It permits reading from a snapshot
+// Txn represents an mvcc transaction (see tikv/storage/doc.go for a definition). It permits reading from a snapshot
 // and stores writes in a buffer for atomic writing.
 type Txn struct {
```

Comment:
can we call it ReqCtx?
There is already a term `txn` for `badger`

Reply:
`MvccTxn`? I don't like using context/ctx because it is pretty meaningless (it's like calling it `RequestThing` or `CommandThing` :-) )

Reply:
alright, `MvccTxn` is better.